.circleci/config.yml

```diff
@@ -19,13 +19,13 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - safecastapi-{{ checksum "Gemfile.lock" }}
-            - safecastapi-
+            - v1-safecastapi-{{ checksum "Gemfile.lock" }}
```

Comment:
why the key change here?

Reply:
Because there was an error like following in [previous build](https://app.circleci.com/pipelines/github/Safecast/safecastapi/946/workflows/682208b1-69f0-4a85-b23e-8d160e054d25/jobs/1451).

```
LoadError: libffi.so.6: cannot open shared object file: No such file or directory - /home/circleci/project/vendor/bundle/ruby/2.6.0/gems/ffi-1.15.0/lib/ffi_c.so
```

Reply:
ah, interesting. Game on then :)